src/routes/index.ts

```diff
@@ -8,6 +8,7 @@ const router = new Router();
 export const registerRoutes = (app: Koa) => {
     router.post('/config', debugUserOpRequest.middleware);
     router.post('/api/v1/:networkId', validateNetworkId, debugUserOpRequest.middleware);
+    router.post('/api/v1/:networkId/supported-entrypoints', validateNetworkId, debugUserOpRequest.middleware);
```

Comment:
Don't need to register any new end point... same end point will work for this. Just need to support new jsonrpc method.